curated_transformers/tokenization/hf_loader.py

```diff
@@ -0,0 +1,76 @@
+from typing import List
+import json
+from spacy.tokens import Doc
+from thinc.api import Model
+
+from .._compat import has_hf_transformers, transformers
+from ..util import registry
+from .bbpe_encoder import ByteBPEProcessor
+from .sentencepiece_encoder import SentencePieceProcessor
+from .wordpiece_encoder import WordPieceProcessor
+
+
+@registry.model_loaders("curated-transformers.HFPieceEncoderLoader.v1")
+def build_hf_piece_encoder_loader(*, name, revision: str = "main"):
+    def load(model: Model, X: List[Doc] = None, Y=None) -> Model:
+        if not has_hf_transformers:
+            raise ValueError("requires transformers package")
+
+        tokenizer = transformers.AutoTokenizer.from_pretrained(name, revision=revision)
+        return _convert_encoder(model, tokenizer)
+
+    return load
+
+
+def _convert_encoder(model: Model, tokenizer: "transformers.PreTrainedTokenizerBase"):
+    if isinstance(tokenizer, transformers.BertTokenizerFast):
+        return _convert_wordpiece_encoder(model, tokenizer)
+    elif isinstance(tokenizer, transformers.RobertaTokenizerFast):
+        return _convert_byte_bpe_encoder(model, tokenizer)
+    elif isinstance(tokenizer, transformers.XLMRobertaTokenizerFast):
+        return _convert_sentencepiece_encoder(model, tokenizer)
+
+    raise ValueError("Loading from this 🤗 tokenizer is not supported")
```

Comment:
```suggestion
    raise ValueError(f"Loading from the '{type(tokenizer}' 🤗 tokenizer is not supported")
```
We can add a bit more context to the error message here.

Reply:
Great idea, fixed!